### ISSUE TITLE
Mobile App Checkout (iOS)

### ISSUE THREAD

**Issue by netcetera-dev** [Mastercard/masterpass-android-sample-app#2]
Hi guys,
I want to integrate masterpass in my iOS app and for test purpose I configured this server and ran it on my local machine. This is what was done according to instruction on github and the official masterpass integration documentation
- created masterpass developer account
- created masterpass merchant account (with invitation code)
- configured this server with appropriate properties (from previous steps) and ran it on my local machine
After this was done I started to testing, first with web checkout and later with checkout from mobile app (iOS) but there are some issues with this project (or maybe I missed something). When I run localhost:8080/initialize.html to test web checkout, Lightbox appear and says “Something went wrong”. Something similar is with mobile checkout, I have success with generating request token and initialising merchant but when I make checkout, app crashed because switchCheckoutResourceResponseRaw variable is empty string and appropriate cast failed. Can anyone help me with this?

**Comment by oceanhuang** [comment]
Hey,

Did you change the checkoutID on this line?
https://github.com/Mastercard/masterpass-android-sample-app/blob/master/Test-Merchant-Server/web/initialize.html#L28

Btw, on the server side, there is a new SDK now. You can check out the documentation here:
https://developer.mastercard.com/documentation/masterpass-merchant-integration#java

**Comment by netcetera-dev** [comment]
Yes, I changed sample checkoutID with my checkoutID. Maybe I should try with the newest SDK as you mentioned to see if it will work.
